calculator/operationCreators.py

```diff
@@ -0,0 +1,18 @@
+import calculator.operations as  ops
+from abc import ABC, abstractmethod
+
+class CreateOperation(ABC):
+    @abstractmethod
+    def factory_method(self, args):
+        pass
+    def calculate(self, args):
+        operation = self.factory_method(args)
+        return operation.calculate()
```

Comment:
```suggestion
        return self.factory_method(args).calculate()
```
Just a little bit cleaner.